src/client/components/modal/Modal.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import "./modal.css";
+function Modal({ closeModal, permited }) {
```

Comment:
`closeModal` is maybe better expressed as an event handler `onClose`.

`permited` doesn't make sense to me, it confuses me. Is it some kind of `modalContent`?